service/src/main/java/uk/gov/hmcts/reform/fpl/validators/DateOfBirthValidator.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateOfBirthValidator {
+
+    public static boolean dateOfBirthIsInFuture(final String dob) {
```

Comment:
That class should operate on correct type. Validator should be responsible for saying Yay / Nay, not doing deserialisation.

Reply:
Have you considered implementing your own constraint annotation? Something similar to https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Past.html?

If it is too time consuming stick to `DateOfBirthValidator` as is.